src/server/redis_connection.cc

```diff
@@ -385,10 +385,17 @@ void Connection::ExecuteCommands(std::deque<CommandTokens> *to_process_cmds) {
 
     auto cmd_s = Server::LookupAndCreateCommand(cmd_tokens.front());
     if (!cmd_s.IsOK()) {
+      auto cmd_name = cmd_tokens.front();
+      if (util::EqualICase(cmd_name, "host:") || util::EqualICase(cmd_name, "post")) {
+        LOG(WARNING) << "A likely HTTP request is detected in the RESP connection, indicating a potential "
+                        "Cross-Protocol Scripting attack. Connection aborted.";
+        Close();
```

Comment:
```suggestion
        EnableFlag(kCloseAsync);
```
Sorry for misleading. Should use the `kCloseAsync` flag to avoid use-heap-after-free error due to `ExecuteCommands` was called inside conn
s read callback.